docs/setup-guide/build_k8s_from_source/setup_2.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+echo "Setup Part 2 started...\n"
+
+echo "Continuing configuring docker..."
+cat <<EOF | sudo tee /etc/docker/daemon.json
+{
+  "exec-opts": ["native.cgroupdriver=systemd"],
+  "log-driver": "json-file",
+  "log-opts": {
+    "max-size": "100m"
+  },
+  "storage-driver": "overlay2"
+}
+EOF
+sudo systemctl daemon-reload
+sudo systemctl restart docker
+echo "[x] Finished setting up docker.\n"
+
+echo "Starting installing Go..."
+cd $HOME
+sudo apt-get -y install make gcc jq
+wget https://redirector.gvt1.com/edgedl/go/go1.17.1.linux-amd64.tar.gz -O go1.17.1.tar.gz
```

Comment:
"go1.17.1" is repeated a few times here. replace these repetition with a variable

Reply:
added the var GO_VERSION